src/Post.js

```diff
@@ -345,7 +345,13 @@ export const ReactionBar = ({
 };
 
 function slugify(s: string): string {
-  return lowerCase(s).replace(/[^A-Za-z0-9-]+/g, '-');
+  return lowerCase(s)
+    .replace(/\s+/g, '-') // Replace spaces with -
+    .replace(/&/g, '-and-') // Replace & with 'and'
+    .replace(/[^\w\-]+/g, '') // Remove all non-word characters
+    .replace(/\-\-+/g, '-') // Replace multiple - with single -
+    .trimStart() // Trim from start of text
+    .trimEnd(); // Trim from end of text
```

Comment:
This covers more bases. `trimStart` and `trimEnd` are part of ES2019 but hopefully that's still okay.

The main reason I wanted to update the original function was because it only replaced things with a `-` which lead to my "Hello world!" being slugified as "hello-world-".